src/Microsoft.AspNetCore.Mvc.Core/Internal/PropertyValueSetter.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
```

Comment:
We actually use this type in two src projects, so kinda need this be to be non-internal.